OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -510,14 +510,14 @@ public static void SetupKickSpectatorsWidget(Widget parent, OrderManager orderMa
 				{
 					Game.LoadWidget(null, "KICK_SPECTATORS_DIALOG", lobby.Get("TOP_PANELS_ROOT"), new WidgetArgs
 					{
-						{ "clientCount", "{0}".F(spectatorCount) },
+						{ "clientCount", $"{spectatorCount}" },
```

Comment:
Redundant format string?

Reply:
We probably want this to be an int, and then change `string clientCount` in `KickSpectatorsLogic` to match.